src/main/java/com/github/alathra/siegeengines/SiegeEngine.java

```diff
@@ -34,55 +34,55 @@
 public class SiegeEngine implements Cloneable {
 
 	// Core variables
-	public SiegeEngineType type;
-	public Boolean enabled;
-	public String id;
-	public String itemName;
-	public List<String> itemLore;
-	public String worldName;
-	public UUID entityId;
-    public Entity entity;
-    public int yOffset;
-    public int xOffset;
-    public int pitchOffset;
-    public int maxFuel;
-    public double placementOffsetY;
-    public float velocityPerFuel;
+	private SiegeEngineType type;
+	private Boolean enabled;
+	private String id;
+	private String itemName;
+	private List<String> itemLore;
+	private String worldName;
+	private UUID entityId;
+    private Entity entity;
+    private int yOffset;
+    private int xOffset;
+    // private int pitchOffset; // Internal - Never Initialized and Defaulted to 0
```

Comment:
@ShermansWorld I found this variable was never initialized though it was used below. Do you want this to be removed for the time being or have a getter and setter added while adding it back on line 230?